apps/frontend/src/components/BlogNav.tsx

```
@@ -0,0 +1,95 @@
+                    onChange={handleChange}
+                    id="search-toggle"
+                    ref={search}
+                    aria-expanded={search?.current?.checked || false}
+                    className="peer/search hidden"
+                />
+                <label
+                    htmlFor="search-toggle"
+                    role="button"
+                    aria-controls="search"
+                    className="flex cursor-pointer items-center rounded-xl bg-gray-200 px-3 py-1 text-xs"
+                >
```

Biome (1.9.4)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)